config.yml

```diff
@@ -8,11 +8,11 @@ gitlab:
   url: https://gitlab.yourcompany.com
   # You can also set in your environment GITLAB_TOKEN
   # this can be both private token OR OAuth2 access token
-  token: "<private token of an admin user>"
+  token: '<private token of an admin user>'
```

Comment:
I think that you pushed the changes to `config.yml` by mistake as they are not related to the app code that you are changing. If I am right then please revert them.

Reply:
I will definitely revert these changes. My apologies, I am such a newb.